README.md

```diff
@@ -14,13 +14,16 @@ Install npm modules.
 ```
 npm install
 ```
-View on localhost.
+Development:
+```
+npm run bundle:watch
 ```
-npm start
 ```
-Run tests.
+npm run dev:start
 ```
-npm test
+Production:
+```
```

Comment:
you need to tell people run the bundle script or change the start script to something like `npm run bundle && node ...etc`